### PR TITLE
Standardize On APPSRE Scripts

### PR DESCRIPTION
**Pull request by dustman9000**
Standardize on app-sre scripts and generate-operator-bundle.py to work with app-sre and boilerplate.

### CODE REVIEW

File: hack/app_sre_create_image_catalog.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+git push origin "$BRANCH_CHANNEL"
+
+popd
+
+# build the registry image
+REGISTRY_IMG="quay.io/app-sre/must-gather-operator-registry"
+DOCKERFILE_REGISTRY="Dockerfile.olm-registry"
+
+cat <<EOF > $DOCKERFILE_REGISTRY
+FROM quay.io/openshift/origin-operator-registry:4.5
```
Comment (csheremeta) [inline hack/app_sre_create_image_catalog.sh:98]: do we want the version specified here?

File: hack/generate-operator-bundle.py [hunk 2]
```diff
@@ -4,153 +4,107 @@
...
+
+csv['spec']['install']['spec']['clusterPermissions'] = []
+
+# Add operator role to the CSV:
+with open('deploy/02_must-gather-operator.ClusterRole.yaml', 'r') as stream:
+    operator_role = yaml.load(stream)
+    csv['spec']['install']['spec']['clusterPermissions'].append(
+        {
+            'rules': operator_role['rules'],
+            'serviceAccountName': OPERATOR_NAME,
```
Comment (csheremeta) [inline hack/generate-operator-bundle.py:79]: I see what this is doing, but how does the SA in 04_must-gather-admin.ServiceAccount.yaml get created?

Comment (dustman9000) [reply hack/generate-operator-bundle.py:79]: My understanding is that OLM ensures that the SA is created ahead of time: https://github.com/operator-framework/olm-book/blob/master/docs/operator-scoping.md#scoping-member-operator-permissions

File: hack/app_sre_create_image_catalog.sh (hunk 1 above)
Comment (dustman9000) [reply hack/app_sre_create_image_catalog.sh]: i've fixed this to use latest

**Comment by csheremeta** [comment]
/lgtm
let's rip out that last commit in a separate PR once things are working (and https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/11761 is merged)

**Pull request merged**